src/System.Linq/src/System/Linq/DebugView.cs

```diff
@@ -97,4 +97,34 @@ public object[] Items
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
         private IEnumerable _enumerable;
     }
+
+    internal sealed class GroupingDebuggerProxy<TKey, TElement>
+    {
+        private readonly Grouping<TKey, TElement> _grouping;
```

Comment:
@OmarTawfik Private fields don't show up for me when I view type proxies in the debugger.

![hiddenprivatemembers](https://cloud.githubusercontent.com/assets/9159214/22388167/261ed84a-e4ad-11e6-9c7d-9aa0381c40c6.PNG)